packages/common/test/search/_internal/hubSearchChannels.test.ts

```diff
@@ -157,4 +163,93 @@ describe("discussionsSearchItems Module |", () => {
       name: ["Foo"],
     } as any as ISearchChannels);
   });
+  it("exclodes group enrichment when include groups not requested", async () => {
```

Comment:
Minor, typo `excludes`

Reply:
Oops, thank you for catching these typos!